### PR TITLE
Creating config.yml file for Welcome bot

### PR DESCRIPTION
**Pull request by malvikasharan**
# Summary

I have opened an issue #132 and adding the config file for the welcome bot

## what should reviewers focus on?

- The messages in the Config file make sense
- They can test this bot on [Turing Way](https://github.com/alan-turing-institute/the-turing-way) repo by creating a new issue
- Configure the welcome bot as described in my issue
- Ask someone who hasn't already created a new issue or PR, can test if the bot works in this repo by creating an issue

### CODE REVIEW

**Comment by Remi-Gau** [comment]
Brilliant. I will assign some reviewers to this.

**Comment by Remi-Gau** [comment]
@cmaumet @llevitis @DorienHuijser 

@malvikasharan has pretty much set up the `welcome bot` for us.  :robot: 

I think this would be a nice way to welcome project submission as issues and also first PR and merge especially on for the venue page #38.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  ![Welcome Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:8ff47a85-7250-4d86-8e48-2f346b48b2c1:BannerWelcome.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  :tada: Welcome to the 2020 OHBM Hackathon! :tada:
+  We're really excited to have your input on this repository and welcome your idea! :sparkling_heart:
+  If you haven't done so already, please make sure you check out our [README file](https://github.com/ohbm/hackathon2020/blob/master/README.md) and [Code of Conduct](https://github.com/ohbm/hackathon2020/blob/master/code_of_conduct.md).
+
```
Comment (DorienHuijser) [inline .github/ISSUE_TEMPLATE/config.yml:13]: Maybe add something like: "Did you issue a new hackathon project? Please check if you have provided all relevant [information] (LINK?) and an image for the [website](https://ohbm.github.io/hackathon2020/hackathon/)."
(if desirable)

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  ![Thank You Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:7fbd97cf-283b-480c-b8e1-11866e26245c:BannerThanks.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  :sparkling_heart: Thanks for opening this pull request! :sparkling_heart:
+  We really appreciates your time and effort to contribute to the project.
```
Comment (DorienHuijser) [inline .github/ISSUE_TEMPLATE/config.yml:22]: appreciates = appreciate

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  Please make sure you have read our [https://github.com/ohbm/hackathon2020/blob/master/README.md) and shared details in your pull request.
```
Comment (DorienHuijser) [inline .github/ISSUE_TEMPLATE/config.yml:23]: Link is broken: "Please make sure you have read our [Readme] (link) ..."
What kind of details do you mean here? "...and shared all relevant details in your pull request."?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  If we don't acknowledge this pull request after 7 days, feel free to comment below.
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  ![Congratulations Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:32fbdb89-ae1b-434e-830c-88ade86724cc:BannerCongratulations.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  Congrats on merging your first pull request! :tada:
+  We here at OHBM team really appreciate your contributions! :sparkling_heart:
```
Comment (DorienHuijser) [inline .github/ISSUE_TEMPLATE/config.yml:34]: contributions = contribution(s)
;)

**Comment by Remi-Gau** [comment]
@DorienHuijser  Looks like you're rocking this review on github thing.

@malvikasharan don't feel like you have to make those edits. We can take it from there if you want. #WeAreAllBusy #TooBusy

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (malvikasharan) [reply .github/ISSUE_TEMPLATE/config.yml:13]: Hey Dorian, You can directly edit the file by using the '+-' symbol. I would leave the editing totally up to you and other folks from the organisation. 🎉  
image

Do call me in if there is some error that need fixing.

**Comment by malvikasharan** [comment]
I will be happy for you all to change the file as it works best for you. It's only a draft to begin this work. I am flying away to another meeting. Do call me in if there are some errors that need fixing.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  ![Welcome Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:8ff47a85-7250-4d86-8e48-2f346b48b2c1:BannerWelcome.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  :tada: Welcome to the 2020 OHBM Hackathon! :tada:
+  We're really excited to have your input on this repository and welcome your idea! :sparkling_heart:
+  If you haven't done so already, please make sure you check out our [README file](https://github.com/ohbm/hackathon2020/blob/master/README.md) and [Code of Conduct](https://github.com/ohbm/hackathon2020/blob/master/code_of_conduct.md).
```
Comment (DorienHuijser) [inline .github/ISSUE_TEMPLATE/config.yml:12]:
```suggestion
  If you haven't done so already, please make sure you check out our [README](https://github.com/ohbm/hackathon2020/blob/master/README.md) and [Code of Conduct](https://github.com/ohbm/hackathon2020/blob/master/code_of_conduct.md).
```

**Comment by Remi-Gau** [comment]
I will also ping the traintrack instructors to see if they want to have an input on this as the welcome bot will be really nice to have if attendees decide to do their PR for the venue page.

**Comment by KirstieJane** [comment]
Hey @DorienHuijser - I'm just going to add this here because you said you're new to this process. Please excuse me if you already know this!

In your review, you've suggested changes (README file --> README for example). Your review then "Requests changes". This workflow is totally fine. It's asking @malvikasharan to make the changes that you've suggested.

In the conversation though, @malvikasharan has asked that you - as a member of the OHBM Hackathon community make the changes yourself. Now, you might not have write access to do that though, so it might be that @Remi-Gau or someone who has write access to the repository needs to make those changes.

Does this make sense? Please ask if not - doing a PR review can be a little jargon-y the first time ✨

**Comment by DorienHuijser** [comment]
@KirstieJane thanks! I wasn't aware that I have to re-review it after changes have been made.  Should I have chosen "comment" or "approve" instead of "request changes"? 
I also see the option to "Commit suggestion" for each change I suggested, would that be an alternative so that there is no re-review required and/or @malvikasharan doesn't have to make the changes themselves?

**Comment by KirstieJane** [comment]
Yes! If you can commit (and @malvikasharan has said that's ok - which she has) then go ahead and commit the changes 🚀 

And yes, approve means approve, and comment means "just a comment". "Request changes" means "block this PR until I approve of the updates".  I think its a super non-intuitive choice of phrasings! It's useful from GitHub from the perspective of maintainers wanting to have lots of control.... I just think its (another) barrier to learning all this stuff!

Thank you for participating!! ✨

**Comment by malvikasharan** [comment]
@DorienHuijser Thank you so much for all the edits! 🎉 They look great! I totally agree with Kirstie, there is a lot to jargons and many non-intuitive Git words. Thank you for patiently looking into this PR, you are totally nailing the GitHub PR review. 🌺

Thank you @KirstieJane for taking the time to explain this.

**Comment by DorienHuijser** [comment]
@KirstieJane thank you for explaining so patiently! :tada: 
@malvikasharan I approved the changes now, hopefully next time will go a bit smoother 😉

**Comment by Remi-Gau** [comment]
Thanks for all those explanations @KirstieJane. I admit that our onboarding process has been a bit "thin" given our time frame. 

But @DorienHuijser you have caught up with all this super fast. :-)

I have made some additional changes linking to some other document that might be more relevant in the hackathon context. @DorienHuijser feel free to amend.

I will ask Liza for a cross-check tomorrow and then we should be ready to go.

**Pull request merged**